shared/src/ledger/events.rs

```diff
@@ -153,7 +162,7 @@ impl From<ProposalEvent> for Event {
 }
 
 /// Convert our custom event into the necessary tendermint proto type
-impl From<Event> for crate::facade::tendermint_proto::abci::Event {
+impl From<Event> for tendermint_proto::abci::Event {
```

Comment:
this should be `crate::facade::tendermint_proto::abci::Event`, right?

Reply:
There is no `facade` module in `shared` (currently)`, though there is something similar in `shared/lib.rs`

https://github.com/anoma/namada/blob/6cbce4a4dcdd3811679f5eb4aa7c82f51dff74e6/shared/src/lib.rs#L9-L16

Reply:
oh right, this isn't in `apps`... yeah, we can use `crate::<dep>` inside of `shared/`

Reply:
Oh yeah, this should be using that, will update